src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/KpiService.kt

```diff
@@ -18,31 +18,33 @@ class KpiService(
   private val offenderSearchService: OffenderSearchService,
 ) {
 
-  fun getNumberOfReviewsConductedAndPrisonersReviewed(day: LocalDate): ReviewsConductedPrisonersReviewed = runBlocking {
```

Comment:
`getNumberOfReviewsConductedAndPrisonersReviewed` was called from inside a coroutine so should avoid using `runBlocking`